### PR TITLE
blurb added to note in authn api

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** blurb added to note in authn api
- **Is this PR related to a Monolith release?** no

### Resolves:

* [OKTA-580251](https://oktainc.atlassian.net/browse/OKTA-580251)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/6db4ab227b8ad09266a460af4a03168f591afb8d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | 71 | 596 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c0ac44f6-ee4f-4742-9a7f-0dfa67c1e459) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.f68a1512-7669-4f00-8561-f5449604257c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/fd8e40df688edf6f274d932e0363105f18f985a7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | 71 | 596 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/998aa86b-1839-4645-8fa1-92dbcacc5707) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.cb234d5f-b599-4391-ae0d-ea79101b6598?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 1]
```diff
@@ -5739,7 +5739,7 @@ Starts a new password recovery transaction for a given user and issues a [recove
 
 > **Note:** A valid `factorType` is required for requests without an API token with administrator privileges. For more information, see [Forgot Password with Trusted Application](#forgot-password-with-trusted-application).
 
-> **Note:** The optional parameter `relayState` can be included as part of the body in the Forgot Password request. `relayState` is a link to a site where the user will be redirected when the password recovery operation completes. The 'relayState' link must point to a trusted origin.
+> **Note:** You can include the optional parameter `relayState` as part of the body in the Forgot Password request. `relayState` is a link to a site where the user is redirected when the password recovery operation completes. The 'relayState' link must point to a trusted origin. The `relayState` parameter works only in Okta Classic Engine orgs.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:5742]:
```suggestion
> **Note:** You can include the optional parameter `relayState` as part of the body in the Forgot Password request. `relayState` is a link to a site where the user is redirected when the password recovery operation completes. The 'relayState' link must point to a trusted origin. The `relayState` parameter is only supported in Okta Classic Engine orgs.
```

**Review by vanngo-okta — APPROVED** [review]
Small suggestion. Otherwise, LGTM 🚢

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/745116b61a05e7ab98d636aea405aaccbb2232f2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | 71 | 596 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0aa2e295-f658-4b04-9aba-5dcd7c96e226) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.0c654651-c6a1-4b25-a2d7-9861f657528c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by shamil-okta — APPROVED** [review]

**Pull request merged**